src/main/java/com/crud/tasks/controller/TaskController.java

```diff
@@ -0,0 +1,38 @@
+package com.crud.tasks.controller;
+
+import com.crud.tasks.domain.TaskDto;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping("/v1/task")
+public class TaskController {
+
+    @RequestMapping(method = RequestMethod.GET, value = "getTasks")
+    public List<TaskDto> getTasks() {
+        return new ArrayList<>();
+    }
+
+    @RequestMapping(method = RequestMethod.GET, value = "getTask")
+    public  TaskDto getTask(Long taskId) {
+        return new TaskDto(1L, "test title", "test_content");
+    }
+
+    @RequestMapping(method = RequestMethod.DELETE, value = "deleteTask")
+    public void deleteTask(long taskId) {
+    }
+
+    @RequestMapping(method = RequestMethod.PUT, value = "updateTask")
+    public TaskDto updateTask(TaskDto taskDto) {
+        return new TaskDto(1L, "Edited test title", "test_content");
+    }
+
+    @RequestMapping(method = RequestMethod.POST, value = "createTask")
+    public void createTask(TaskDto taskDto) {
```

Comment:
przy tworzeniu ja bym zwracał DTO - bo dzięki temu jeśli ID jest kluczem z sekwencji to możemy od razu dodać obiekt w aplikacji front-endowej bez pełnego przeładowania listy.